### ISSUE TITLE
Clarification on instructions for authors

### ISSUE THREAD

**Issue by mkalz** [theResearchMethodBook/chapter-template#2]
I need two clarifications:

1. What is meant with examples in the chapters? Are these exercises?
2. Since I do not want to overwrite the chapter-template with a commit I ask myself if I should branch the chapter template.

**Comment by phish108** [comment]
> What is meant with examples in the chapters? Are these exercises?

@marcuspecht suggested examples in terms of worked out examples. This needs clarification. 

> Since I do not want to overwrite the chapter-template with a commit I ask myself if I should branch the chapter template.

Nobody will work on the template. 

This repo serves as a template to populate the actual chapter repos with all bells and whistles pre-configured. 

The only ones who are working on it is us, when improving the authoring experience. 

I imagine the process as following: 

1. An author suggests a new chapter in the `book` repo by creating an issue in that repo. 
2. Once we accept the chapter we create the chapter repo for the authors from the chapter template. 
3. We copy the chapter information to the respective fields  in the `chapter.qmd` file. 
4. The authors create their chapter in their now isolated repo.
5. Once the chapter is ready for release, we weave the chapter into the actual book.

I am working on an Editor guide and an Author guide. See theResearchMethodBook/book#4, theResearchMethodBook/book#5 and theResearchMethodBook/book#6

**Comment by phish108** [comment]
I created a new issue form in the book repo.

**Comment by mkalz** [comment]
Thanks - gotcha! I would still propose to add the apa.csl to the template folder and not to the book though.

**Comment by phish108** [comment]
@mkalz i included and configured the apa.csl to all repos. It is available in the ˋ_contribˋ folder. By convention this folder includes all tools that are not developed by a project directly.

you may want to pull the latest version to see these updates locally.

**Comment by mkalz** [comment]
Thanks for clarification!
